go/client/cmd_simplefs_list.go

```diff
@@ -95,28 +198,58 @@ func (c *CmdSimpleFSList) Run() error {
 		if err != nil {
 			return err
 		}
+		gotList := false
 		for {
 			listResult, err := cli.SimpleFSReadList(ctx, opid)
+			// Eat the error here because it may just mean the results
```

Comment:
my guess is that `listResult` is empty the last time through this loop. And you should exit there. My guess is that solves two problems: (1) it simplifies the exit code below (an error is always an error and it allows you to get rid of the TODO); and (2) it explains where the empty line is coming from, which is running `output` on an empty list.